.github/workflows/firebase-hosting-merge-yml

```diff
@@ -0,0 +1,19 @@
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on merge
+'on':
+  push:
+    branches:
+      - main
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: '${{ secrets.GITHUB_TOKEN }}'
+          firebaseServiceAccount: '${{ secrets.FIREBASE_SERVICE_ACCOUNT_ZYMPL_LEEWAY }}'
+          channelId: live
+          projectId: zympl-leeway
```

Comment:
Change the project id to your project id. you can find it in the `.github/workflows/firebase-hosting-pull-request.yml` file

Reply:
myportfolio-adhithyarajan